stdlib/_thread.pyi

```diff
@@ -16,11 +16,16 @@ class LockType:
     def acquire(self, blocking: bool = True, timeout: float = -1) -> bool: ...
     def release(self) -> None: ...
     def locked(self) -> bool: ...
+    def acquire_lock(self, blocking: bool = True, timeout: float = -1) -> bool: ...
+    def release_lock(self) -> None: ...
+    def locked_lock(self) -> bool: ...
```

Comment:
I would group those differently:

```py
def acquire(...)
def acquire_lock(...)
``` 

(The `*_lock` methods are exactly the same as the non-`*_lock` and are legacy methods).

Reply:
I grouped them together same as in the PR that added these methods to `Lock`:

https://github.com/python/typeshed/blob/d2ccc9a723f1a38d234d29ff5519be295e68e25f/stdlib/threading.pyi#L112-L114

Reply:
Ah yes, my bad. (Maybe add the "undocumented" comment then?)